src/Analyzers/CSharp/Analyzers/InvokeDelegateWithConditionalAccess/InvokeDelegateWithConditionalAccessAnalyzer.cs

```diff
@@ -84,13 +82,13 @@ private void SyntaxNodeAction(SyntaxNodeAnalysisContext syntaxContext)
                 innerStatement = block.Statements[0];
             }
 
-            if (!innerStatement.IsKind(SyntaxKind.ExpressionStatement, out ExpressionStatementSyntax expressionStatement))
+            if (!innerStatement.IsKind(SyntaxKind.ExpressionStatement, out ExpressionStatementSyntax? expressionStatement))
             {
                 return;
             }
 
             // Check that it's of the form: "if (a != null) { a(); }
-            if (!(expressionStatement.Expression is InvocationExpressionSyntax invocationExpression))
+            if (expressionStatement.Expression is not InvocationExpressionSyntax invocationExpression)
```

Comment:
Lots of noise going on in this review. It's already large, so this makes it tremendously more time consuming to figure out which parts are actually needed and which parts are just change for the sake of change.